### ISSUE TITLE
Markdown links with only numbers and special characters in link tag do not render

### ISSUE THREAD

**Issue by shbhadra** [microsoft/AdaptiveCards#3000]
# Platform
* Android
* iOS

# Author or host
* Host

# Version of SDK
* Android : 1.1.1
* iOS : 1.0.3

# Issue
Markdown links that have a combination of numbers and special characters only do not get rendered. Consider the following JSO

```
{

  "contentType": "application/vnd.microsoft.card.adaptive",

  "type": "AdaptiveCard",

  "body": [

    {

      "size": "medium",

      "text": "Build [20190523.2](https:www.google.com) failed",

      "weight": "bolder",

      "type": "TextBlock",

      "wrap": true,

      "isSubtle": false,

      "spacing": "medium",

      "color": "default",

      "horizontalAlignment": "left",

      "separator": true

    }

  ],

  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",

  "version": "1.0"

}  
```

The link inside the "text" field of the TextBlock is not rendered. However if we change the link tag to remove the special character or add an alphabet, the link is rendered fine. Thus
`"text": "Build [201905232](https:www.google.com) failed",`
or 
`"text": "Build [Some text 20190523.2](https:www.google.com) failed",`
are rendered fine.
The same problem is also there with links in factSet and column set.

**Comment by jjustinms** [comment]
@shalinijoshi19 @nesalang  will this be backported to 1.2?

**Comment by shalinijoshi19** [comment]
Hey @jjustinms, we actually didn't realize that this was something that needed to be patched 1.2 with. Are there more issues that you are tracking at your end that you would like for us to port over to 1.2? AT this point for our patches we are being very specific and only porting needed bug fixes. We just wrapped up a 1.2.1 with some Teams-impacting bug fixes, and can send up a 1.2.2 for iOS + Android with this and any other fixes you are tracking that have been fixed and need to be backported. @nesalang let's come up with a timeline for the 1.2.2 - -say end of this week?
